.env.example

```
@@ -6,4 +6,4 @@ VMDR_FINDING_TYPES=Potential,Confirmed
 MIN_SCANNED_SINCE_DAYS=
 MIN_FINDINGS_SINCE_DAYS=
 INGEST_WEB_APP_SCANS=true
-DETECTION_RESULTS_QIDS=1,2,3
+VMDR_FINDING_RESULT_QIDS=1,2,3
```

I think it would be good to default this to the same value documented in the instance config here: https://github.com/JupiterOne/graph-qualys/blob/d026da38f4963b751283c09dc7a2b0f2b4c91121/src/instanceConfigFields.ts#L112
```suggestion
VMDR_FINDING_RESULT_QIDS=
```